job_templates/packaging_job.xml.em

```diff
@@ -35,6 +34,11 @@
             </a>
           </choices>
         </hudson.model.ChoiceParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>CI_AMENT_BUILD_ARGS</name>
+          <description>Additional arguments passed to 'ament build'.</description>
+          <defaultValue>@(ament_build_args_default)</defaultValue>
+        </hudson.model.StringParameterDefinition>
```

Comment:
example of what the jobs configuration page would look like:
http://ci.ros2.org/job/ci_packaging_linux-aarch64/build?delay=0sec
I havent deployed the regular ci jobs but they would look the same (CI_AMENT_BUILD_ARGS before any parameter that is job specific v.s. at the bottom before). Is it ok for everyone?